lib/prepare.js

```diff
@@ -236,14 +236,14 @@ async function genComponentRegistrationFile ({ sourceDir }) {
   return `import Vue from 'vue'\n` + components.map(genImport).join('\n')
 }
 
-const indexRE = /(?<=(^|\/))(index|readme)\.md$/i
+const indexRE = /^\/?([^/]+\/)*(index|readme)\.md$/i
```

Comment:
This may help:
```js
const indexRE = /(^\/?([^/]+\/)*)(index|readme)\.md$/i
```

Reply:
And maybe this is enough:
```js
const indexRE = /(^|.*\/)(index|readme)\.md$/i
```

Reply:
The second one looks good, I just tested for it, so let me leverage it.